CMakeLists.txt

```diff
@@ -1,5 +1,25 @@
 cmake_minimum_required(VERSION 3.20)  # For using CMAKE_<LANG>_BYTE_ORDER
 
+if(NOT DEFINED USE_VCPKG OR USE_VCPKG)
+  message("NOT DEFINED USE_VCPKG OR USE_VCPKG")
+  if(DEFINED ENV{VCPKG_ROOT})
+    message("DEFINED ENV{VCPKG_ROOT}")
+    if (CMAKE_TOOLCHAIN_FILE)
```

Comment:
When is this variable set? Only for cross-compilation builds?

Reply:
see https://github.com/DescentDevelopers/Descent3/pull/469#discussion_r1673465920